enterprise_access/apps/subsidy_access_policy/models.py

```diff
@@ -437,16 +437,34 @@ def get_list_price(self, lms_user_id, content_key):  # pylint: disable=unused-ar
 
     def aggregates_for_policy(self):
         """
-        Returns aggregate transaction data for this policy.
+        Returns aggregate transaction data for this policy. The result is cached via ``RequestCache``
+        for other use in the scope of a single request.
 
         Raises:
             requests.exceptions.HTTPError if the request to Subsidy API fails.
         """
+        _cache = request_cache(namespace=REQUEST_CACHE_NAMESPACE)
+        cache_key = versioned_cache_key('aggregates_for_policy', self.subsidy_uuid, self.uuid)
```

Comment:
Should we also mix in a request ID or something else that would ensure this cache is only used for the duration of the session?

Reply:
The `RequestCache` class achieves that affect by design (using a thread-local to store cached values). https://github.com/openedx/edx-django-utils/tree/master/edx_django_utils/cache#requestcache